frontend-react/src/components/Admin/OrgsTable.tsx

```diff
@@ -39,7 +71,7 @@ export function OrgsTable() {
             memberType === MemberType.SENDER ||
             memberType === MemberType.PRIME_ADMIN
         ) {
-            payload.senderName = senderName || "default";
+            payload.service = service || "default";
```

Comment:
For the sender sides of things, we have these "default" senders everywhere. I'm going to make a pass and see if these are creating any issues. As the admin info banner states, only current known break could be for admins mocking incorrectly before navigating to Daily Data (break = ErrorBoundary, not full on app break)

Reply:
The only places on the site we're using sender names are in the SenderModeBanner and the FileHandler. I've removed this "default" fallback, meaning the same will be true for both Daily Data and Validation for admins (and the banner on the admin settings page will reflect this):

- Daily Data requires mocking MemberType.RECEIVER from settings
- Validation requires mocking MemberType.SENDER from settings

When you mock the given member type, your active membership will attempt to fetch services based your new designation. This hopefully relieves the front-end of _any_ control over what those values are and uses the settings table through the API to get that info exclusively.


<img width="199" alt="Screen Shot 2022-10-26 at 4 08 52 PM" src="https://user-images.githubusercontent.com/4022304/198156092-f9585053-2cb2-49fe-aeac-21d959b201ae.png">
<img width="206" alt="Screen Shot 2022-10-26 at 4 08 41 PM" src="https://user-images.githubusercontent.com/4022304/198156090-d38ce1b5-8673-4a89-948c-754e887eae75.png">